packages/core/src/web/helpers/monitor-status.ts

```diff
@@ -5,10 +5,12 @@ import type { IReport } from '@core/interfaces/IDevice';
 export enum ButtonTypes {
   DISABLED_PAUSE = 4,
   DISABLED_PLAY = 2,
+  DISABLED_RESUME = 8,
   DISABLED_STOP = 6,
   NONE = 0,
   PAUSE = 3,
   PLAY = 1,
+  RESUME = 7,
   STOP = 5,
 }
```

Comment:
Can you help add `/* eslint-disable perfectionist/sort-enums */` and `/* eslint-enable perfectionist/sort-enums */` around this enum